spotbugs/src/main/java/edu/umd/cs/findbugs/Version.java

```diff
@@ -99,22 +105,22 @@ public static void main(String[] argv) throws InterruptedException {
         String arg = argv[0];
 
         if ("-release".equals(arg)) {
-            System.out.println(VERSION_STRING);
+            LOGGER.info(VERSION_STRING);
```

Comment:
I've never used JUL API: is the INFO level enabled by default?

Reply:
It's configurable in `jre/lib/logging.properties`, but yes, the default level in that file is INFO.